index.css

```diff
@@ -426,6 +492,19 @@ textarea {
   margin-block-start: 0;
 }
 
+.d-none {
+  display: none;
+}
+
+.h-100 {
+  height: 100vh;
+}
+
+.no-scroll-bg {
+  overflow-y: hidden;
+  background: none;
+}
+
 /* desktop view */
```

Comment:
_Good Job and documentation, beautiful project, your project's been approved, you can merge now, congrats ✔✨_

💪😊Keep up the good work📌